.travis.yml

```diff
@@ -1,20 +1,23 @@
 language: android
 jdk: oraclejdk8
 sudo: required
+
 env:
   global:
   - GRADLE_OPTS="-Xmx512m"
   matrix:
- # - ADB_INSTALL_TIMEOUT=15 ANDROID_BUILD_TOOLS_VERSION=24.0.2 ANDROID_TARGET=android-23 ANDROID_ABI=armeabi-v7a
-  - ADB_INSTALL_TIMEOUT=15 ANDROID_BUILD_TOOLS_VERSION=23.0.2 ANDROID_API_LEVEL=23 ANDROID_ABI=armeabi-v7a ANDROID_TAG=google_apis
+  - ADB_INSTALL_TIMEOUT=15 ANDROID_BUILD_TOOLS_VERSION=25.0.3 ANDROID_TARGET=android-25 ANDROID_ABI=armeabi-v7a
```

Comment:
Any reasons to remove `ANDROID_TAG` value as this is used [here](https://github.com/fossasia/loklak_wok_android/pull/31/files#diff-354f30a63fb0907d4ad57269548329e3R25)?